### PR TITLE
Publishers

### PR DESCRIPTION
**Pull request by tarasborov**
The ros publishers were written. The PID controllers were tuned on slow dynamics at first in order to avoid unstable system.

### CODE REVIEW

File: rs_control/scripts/listeners.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    rospy.Subscriber("/spot/joint_front_left_hip_y_controller/state", JointControllerState, callback_front_left_hip_y)
+    rospy.Subscriber("/spot/joint_front_left_knee_controller/state", JointControllerState, callback_front_left_knee)
+
+    rospy.Subscriber("/spot/joint_front_right_hip_x_controller/state", JointControllerState, callback_front_right_hip_x)
+    rospy.Subscriber("/spot/joint_front_right_hip_y_controller/state", JointControllerState, callback_front_right_hip_y)
+    rospy.Subscriber("/spot/joint_front_right_knee_controller/state", JointControllerState, callback_front_right_knee)
+
+    rospy.Subscriber("/spot/joint_rear_left_hip_x_controller/state", JointControllerState, callback_rear_left_hip_x)
+    rospy.Subscriber("/spot/joint_rear_left_hip_y_controller/state", JointControllerState, callback_rear_left_hip_y)
+    rospy.Subscriber("/spot/joint_rear_left_knee_controller/state", JointControllerState, callback_rear_left_knee)
```
Comment (ldemk) [inline rs_control/scripts/listeners.py:71]: hardcoded topic are evel.
Use yaml or launch file to pass them as arguments

File: rs_control/scripts/listeners.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    rospy.loginfo(rospy.get_caller_id() + "I heard %s", data.process_value)
+
+
+def listener():
+    # In ROS, nodes are uniquely named. If two nodes with the same
+    # name are launched, the previous one is kicked off. The
+    # anonymous=True flag means that rospy will choose a unique
+    # name for our 'listener' node so that multiple listeners can
+    # run simultaneously.
+    rospy.init_node('listener', anonymous=True)
```
Comment (ldemk) [inline rs_control/scripts/listeners.py:60]: do not hardcore names - this is wrong

**Comment by ldemk** [comment]
@tarasborov confirm that you have read my comments and we can merge this PR for now

**Comment by tarasborov** [comment]
Yes, I have read them.

**Review by ldemk — APPROVED** [review]

**Pull request merged**